### ISSUE TITLE
reachingvideo1DeepCut_resnet101_MouseMay9shuffle1_12000.h5 does not exist

### ISSUE THREAD

**Issue by leiyaohui** [DeepLabCut/DeepLabCut#283]
**Your Operating system and DeepLabCut version**
win10  with an Anaconda Env,DeepLabCut 2.0.6.2
Please state your operating system, env, and which version of DeepLabCut you are using.
Example: Ubuntu 16.04 LTS, with an Anaconda Env, & DeepLabCut1.x or 2.x.

**If applicable, please also complete the following information:**
 
**Describe the bug**
A clear and concise description of what the bug is!
hello！This error occurred when I ran deeplabcut. analyze_videos (path_config_file, videofile_path, videotype='. MTS').
File "D:\Anaconda\envs\dlc-windowsGPU\lib\site-packages\deeplabcut\pose_estimation_tensorflow\predict_videos.py", line 269, in AnalyzeVideo
    pd.read_hdf(dataname)
  File "D:\Anaconda\envs\dlc-windowsGPU\lib\site-packages\pandas\io\pytables.py", line 347, in read_hdf
    'File %s does not exist' % path_or_buf)
FileNotFoundError: File F:\DeepLabCut-master\examples\reachingvideo1DeepCut_resnet101_MouseMay9shuffle1_12000.h5 does not exist
I want to know where this file was generated and why I didn't.
**To Reproduce**

**Expected behavior**

**Screenshots**
![image](https://user-images.githubusercontent.com/42469241/57497426-38b89980-730a-11e9-96eb-8a4a8bcb1d04.png)

**Additional context**
I hope to get your help. Thank you.

**Comment by MMathisLab** [comment]
Seems either your config path video file path is misspelled. You may also want to try with another video type; i’m not sure openCV supports .MTS, perhaps convert the video to .mp4, but carefully check the paths first.

**Comment by leiyaohui** [comment]
Thank you for your quick reply. I don't know where I misspelled. There seems to be no mistake after path checking.MTS Video File is OK. I tried it before and it succeeded. Thank you again.

**Comment by MMathisLab** [comment]
The file is created in dlc-models, is the file there?

**Comment by MMathisLab** [comment]
perhaps also related to ffmpeg installation; #45 #54 #273. Namely, please double check you have ffmpeg correctly installed. Inside your env run ``conda install -c conda-forge ffmpeg``

**Comment by leiyaohui** [comment]
Thank you for your reply. I reinstalled it as you suggested, but the problem still bothers me. I tried a lot of video analysis that had similar errors, but with reaching video1. avi analysis, this error did not occur and ran smoothly. In the main folder, two files needed for the above errors were also generated.
![image](https://user-images.githubusercontent.com/42469241/57576945-cfb65a80-749e-11e9-9614-895912a73005.png)
I don't think there should be such a mistake with other videos, because these files were generated later
 What happened? I hope to get your reply. Thank you.
